README.md

```diff
@@ -1 +1,191 @@
-# rerender-vs-surgical-updates
+# Overview
+
+In this example we have implemented using React, Solidjs and Svelte a demo similar to the one they did in the 2021 React Conf presenting React Forget.
+
+## React
+
+Here we have a list of products where we can add a new one or update it by marking it as fav or unfav. Notice that all products are re-rendered even if we don't update their properties. 
```

Comment:
Let's display a list of products and let the users:
- Add new ones.
- Update them by marking them as fav or unfav.

Since we are not implementing opitmizations, all products are re-rendered even if we don't update their properties.